packages/side-nav/package.json

```diff
@@ -0,0 +1,22 @@
+
+  {
+    "name": "@leafygreen-ui/side-nav",
+    "version": "1.0.0",
```

Comment:
nit, I believe we have to have this be initialized to 0.9.0 so that the changelog script correctly runs the first time we use it. Gonna let @brookescarlett weigh in.